examples/postgres/test_all

```diff
@@ -1,7 +1,22 @@
 #!/bin/sh
 set -e
 
-export DATABASE_URL="postgres://localhost/diesel_examples"
+set -a
+. ../../.env || true
+DATABASE_URL=${DATABASE_URL-${PG_DATABASE_URL}}
+set +a
+
+trap finish EXIT
+
+function finish {
+  # Make sure when we're done, we leave behind an empty database. Otherwise,
+  # at the time of writing, running diesel_tests stumbles as it tries to run
+  # conflicting migrations on an overlapping schema.
+  #
+  # TODO: Find a better way to avoid mixing of resources across tests.
+  ../../bin/diesel database drop
+  ../../bin/diesel database touch
```

Comment:
Actually I'm not sure I fully understand the issue you ran into. We hard code the database for examples [here](https://github.com/diesel-rs/diesel/blob/master/examples/postgres/test_all#L4). My point was that we just need to make sure that it doesn't conflict with the example URL given in the CONTRIBUTING document, but it already doesn't. Something else is wrong if the examples were running against your test database.